pkg/logcli/query/query.go

```diff
@@ -427,7 +427,7 @@ func getLatestConfig(client chunk.ObjectClient, orgID string) (*config.SchemaCon
 	if err != errNotExists {
 		return nil, err
 	}
-	return nil, errNotExists
+	return nil, errors.New("could not find a schema config file matching any of the known patterns. First verify --org-id is correct. Then check the root of the bucket for a file with `schemaconfig` in the name. If no such file exists it may need to be created or re-synced from the source.")
```

Comment:
```suggestion
	return nil, errors.New("Could not find a schema config file matching any of the known patterns. First verify --org-id is correct. Then check the root of the bucket for a file with `schemaconfig` in the name. If no such file exists it may need to be created or re-synced from the source.")
```

Reply:
@JStickler In Go, error messages are supposed to start with lower case :/  and it's enforced by the code linter.

@slim-bean However, since the message is directly displayed to the user, we should add a linter exception and start the sentence with uppercase.

Reply:
TIL